### ISSUE TITLE
0.1.2 loading组件样式缺少weui_mask_transparent

### ISSUE THREAD

**Issue by wg5945** [airyland/vux#381]
### Version
0.1.2
### Android or iOS
ALL

### Your Codes 

``` html
<style lang="less">
@import '../../styles/weui/widget/weui_tips/weui_toast';
</style>
```
这个里面缺少
``` html
 .weui_mask_transparent{position:fixed;z-index:1000;width:100%;height:100%;top:0;left:0}
```

**Comment by airyland** [comment]
Thx. Fixed on v0.1.3-rc3

**Comment by xiaolongyuan** [comment]
Loading 挤在一起


![image](https://cloud.githubusercontent.com/assets/3142363/17019240/ea007758-4f6d-11e6-8aa8-176dae42b4b4.png)
